### DISCUSSION TITLE
Custom images for the accounts in ente.io authenticator. When a code is for some unknown company authentication, it leaves it blank.

### DISCUSSION THREAD

**Discussion by DrakerSheriff** [ente-io/ente#4002] [Enhancements]
The title. It would be better to put one of our own image to the 2fa code account.

**Comment by mallomar** [comment]
Seconded. One of the only things Authy does better.

**Reply by DrakerSheriff** [reply]
but even authy doesn't have custom images but they do have the option to search and whatever comes you can put it as the account logo.
